test/conftest.py

```diff
@@ -0,0 +1,95 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+from datetime import datetime
+from pathlib import Path
+from test.tracing.utils import SpyingTracer
+from typing import Generator
+from unittest.mock import Mock, patch
+
+import pytest
+from openai.types.chat import ChatCompletion, ChatCompletionMessage
+from openai.types.chat.chat_completion import Choice
+
+from haystack import tracing
+from haystack.testing.test_utils import set_all_seeds
+
+set_all_seeds(0)
+
+
+@pytest.fixture()
+def mock_tokenizer():
+    """
+    Tokenizes the string by splitting on spaces.
+    """
+    tokenizer = Mock()
+    tokenizer.encode = lambda text: text.split()
+    tokenizer.decode = lambda tokens: " ".join(tokens)
+    return tokenizer
+
+
+@pytest.fixture()
+def test_files_path():
+    return Path(__file__).parent / "test_files"
+
+
+@pytest.fixture
+def mock_chat_completion():
```

Comment:
Is this part of the PR for a specific reason?

Reply:
Ah, no. Removed.